cmds/core/tee/tee_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+func TestTee(t *testing.T) {
+	tempDir := t.TempDir()
+
+	tests := []struct {
+		input  string
+		args   []string
+		append bool
+	}{
+		{
+			"hello",
+			[]string{filepath.Join(tempDir, "a1"), filepath.Join(tempDir, "a2")},
+			false,
+		},
+		{
+			"a\nb\n",
+			[]string{filepath.Join(tempDir, "b1")},
+			true,
+		},
+	}
+
+	for _, test := range tests {
+		var stdout bytes.Buffer
+		var stderr bytes.Buffer
+		cmd := New(test.append, false, test.args)
+		cmd.stdin = strings.NewReader(test.input)
+		cmd.stdout = &stdout
+		cmd.stderr = &stderr
+		err := cmd.run()
+		if err != nil {
+			t.Error(err)
```

Comment:
Should this be `t.Fatalf`? No sense in checking the file contents (line 47+) if an error was returned, right?

Reply:
done